src/components/notebooks/card.tsx

```diff
@@ -1,32 +1,55 @@
-
 export default function Card(props: Notebook) {
-  const title = props.title || '';
-  const description = props.description || '';
-  // TODO: Fetch notebook 'Last Updated' when implemented on backend
-  const lastUpdated = '';
-  const href = `notebooks/${props.id}/`
+  const title = props.title || "";
+  const description = props.description || "";
+
+  // Formats the received date for UI
+  // Adds '0' to single digit date parts
+  const padNumber = (num: number) => {
```

Comment:
Using function is always preferred to arrow function/lambdas to use as function; use arrow function when the function is simple, don't require it's own binding or is bound to the context only. 

This would also mean that you'd need to extract it from the Card component function. 

Here's the equivalent of the arrow function as a function.  
```js
function padNumber(num: number) {
    ...
    return ...
}
```